{"uid":"e3bed70d201ba788","name":"Board Creation","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_board_creation","historyId":"7a20b967a411d7d4449c81c8bbcd2d3a","time":{"start":1734113582734,"stop":1734113620327,"duration":37593},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f6098e187c0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"01769ccf3a648f4e8ad7f6a6ff5abd0b\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"01769ccf3a648f4e8ad7f6a6ff5abd0b\")>\nmethod = <function url_matches.<locals>._predicate at 0x7f6097d1cdc0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f6098e187c0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"01769ccf3a648f4e8ad7f6a6ff5abd0b\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC002\n    @allure.title('Board Creation')\n    def test_board_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:97: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:2082: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1446: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1589: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1599: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1661: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:954: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1093: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1085: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:929: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:668: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:373: TypeError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1734113582468,"stop":1734113582733,"duration":265},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1734113582733,"stop":1734113582733,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f6098e187c0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"01769ccf3a648f4e8ad7f6a6ff5abd0b\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"01769ccf3a648f4e8ad7f6a6ff5abd0b\")>\nmethod = <function url_matches.<locals>._predicate at 0x7f6097d1cdc0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f6098e187c0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"01769ccf3a648f4e8ad7f6a6ff5abd0b\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC002\n    @allure.title('Board Creation')\n    def test_board_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:97: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:2082: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1446: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1589: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1599: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1661: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:954: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1093: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1085: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:929: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:668: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:373: TypeError","steps":[{"name":"Log in to Trello","time":{"start":1734113592745,"stop":1734113620327,"duration":27582},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 97, in test_board_creation\n    self.test_login(driver)\n  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 2082, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1446, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1589, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1599, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1661, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 954, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1093, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1085, in emit\n    msg = self.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 929, in format\n    return fmt.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 668, in format\n    record.message = record.getMessage()\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 373, in getMessage\n    msg = msg % self.args\n","steps":[{"name":"Navigate to the Trello login page","time":{"start":1734113592745,"stop":1734113596037,"duration":3292},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1734113596037,"stop":1734113620257,"duration":24220},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 2082, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1446, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1589, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1599, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1661, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 954, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1093, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1085, in emit\n    msg = self.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 929, in format\n    return fmt.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 668, in format\n    record.message = record.getMessage()\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 373, in getMessage\n    msg = msg % self.args\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":2,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"525f2c551f66d6f6","name":"log","source":"525f2c551f66d6f6.txt","type":"text/plain","size":63294}],"parameters":[],"stepsCount":3,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1734113620495,"stop":1734113620567,"duration":72},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC002"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az1756-422"},{"name":"thread","value":"2849-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":1,"skipped":0,"passed":24,"unknown":0,"total":27},"items":[{"uid":"ebe7a976f9bbb8ba","status":"passed","time":{"start":1734050759846,"stop":1734050792254,"duration":32408}},{"uid":"b712f522d009fdee","status":"passed","time":{"start":1734027356495,"stop":1734027388998,"duration":32503}},{"uid":"fbe3546f571d49d4","status":"passed","time":{"start":1733973727495,"stop":1733973759903,"duration":32408}},{"uid":"fe173ab241eaf438","status":"passed","time":{"start":1733972593173,"stop":1733972625541,"duration":32368}},{"uid":"d099a8c54a381a3f","status":"passed","time":{"start":1733971627803,"stop":1733971660172,"duration":32369}},{"uid":"f80b51a6cde163f3","status":"passed","time":{"start":1733970373428,"stop":1733970405697,"duration":32269}},{"uid":"e93af23a1d7919fb","status":"passed","time":{"start":1733963271666,"stop":1733963304379,"duration":32713}},{"uid":"b6ffceb93316f0ec","status":"passed","time":{"start":1733932543217,"stop":1733932575875,"duration":32658}},{"uid":"1f6133f3933fdea8","status":"passed","time":{"start":1733891386630,"stop":1733891418776,"duration":32146}},{"uid":"af29590663b7c56d","status":"passed","time":{"start":1733890942449,"stop":1733890974677,"duration":32228}},{"uid":"ae98a00f643de8e1","status":"passed","time":{"start":1733890371601,"stop":1733890403921,"duration":32320}},{"uid":"b1c2d5608c775e78","status":"passed","time":{"start":1733889750258,"stop":1733889782322,"duration":32064}},{"uid":"6ae7500196f2992","status":"passed","time":{"start":1733888606463,"stop":1733888638845,"duration":32382}},{"uid":"df6d88ce3e5bd57f","status":"passed","time":{"start":1733843154854,"stop":1733843187353,"duration":32499}},{"uid":"9ef7b1309a4ef52b","status":"passed","time":{"start":1733842618656,"stop":1733842651229,"duration":32573}},{"uid":"8d62caee5f6ae0f7","status":"failed","statusDetails":"AssertionError: assert 'new_board' == 'new_board2'\n  \n  - new_board2\n  ?          -\n  + new_board","time":{"start":1733841772592,"stop":1733841804858,"duration":32266}},{"uid":"58a30db31cea4e8a","status":"failed","statusDetails":"AssertionError: assert 'new_board' == 'new_board2'\n  \n  - new_board2\n  ?          -\n  + new_board","time":{"start":1733841417917,"stop":1733841451655,"duration":33738}},{"uid":"cb00293daf856968","status":"passed","time":{"start":1733840192731,"stop":1733840225082,"duration":32351}},{"uid":"c8fbebf1345eaa32","status":"passed","time":{"start":1733797141513,"stop":1733797171623,"duration":30110}},{"uid":"2f84b3a4a90b1edb","status":"passed","time":{"start":1733586023407,"stop":1733586053091,"duration":29684}}]},"tags":["TC002","TC000"]},"source":"e3bed70d201ba788.json","parameterValues":[]}